docs/tutorials/broadcast.md

```diff
@@ -69,3 +69,17 @@ main();
     > The signature you provided gives this app permission to read and send XMTP messages on your behalf. The signature is securely stored and accessed only to execute your workflows. Click **Delete signature** to revoke this permission and delete the signature from storage.
 
     ![Signature storage disclosure and delete after connection](/img/sig-store-disclosure-delete.png)
+
+## Enable users to opt-in to receive broadcast messages
+
+Before sending broadcast messages to a user, ensure they've agreed to receive the messages.
+
+For example, during onboarding, let the user know that by connecting their wallet account to your app, they are agreeing to receive messages from you. Here is some example text you can build upon:
+
+> By connecting your wallet account with this app, you are agreeing to receive broadcast messages from &lt;your app name&gt;. These messages may include updates and promotions.
+>
+> You can unsubscribe from broadcast messages by replying with `STOP`.
```

Comment:
Is this overly prescriptive, or helpful? Should we provide guidance about how to build this?